pennylane/labs/vibrational/pes_generator.py

```diff
@@ -15,30 +15,27 @@
 per normal modes on a grid."""
 
 import itertools
-import subprocess
+from pathlib import Path
 
 import numpy as np
 
 import pennylane as qml
 from pennylane.data.base._lazy_modules import h5py
-
-from .localize_modes import localize_normal_modes
-from .vibrational_class import (
-    VibrationalPES,
+from pennylane.qchem import VibrationalPES, localize_normal_modes, optimize_geometry
+from pennylane.qchem.vibrational.vibrational_class import (
     _get_dipole,
     _harmonic_analysis,
     _single_point,
-    optimize_geometry,
 )
 
-# pylint: disable=too-many-arguments, too-many-function-args, c-extension-no-member
+# pylint: disable=too-many-arguments, too-many-function-args, c-extension-no-member, too-many-branches
 # pylint: disable= import-outside-toplevel, too-many-positional-arguments, dangerous-default-value
 
-
 # constants
 HBAR = 6.022 * 1.055e12  # (amu)*(angstrom^2/s)
 C_LIGHT = 3 * 10**8  # m/s
 BOHR_TO_ANG = 0.5291772106  # factor to convert bohr to angstrom
+AU_TO_CM = 219475  # factor to convert Hartree to cm^-1
```

Comment:
Hey @ddhawan11, just a quick fly-by request: can all of these be expressed in terms of the standardised values provided by `scipy.constants` rather than hard-coded values ? https://docs.scipy.org/doc/scipy/reference/constants.html

Reply:
Changed this to scipy constants. There are values where we needed to change units, added a todo to make sure everything works in atomic units before we move it to pennylane.